components/activityTemplate/Header/ActivityHeader.js

```diff
@@ -62,7 +62,9 @@ export default function ActivityHeader(props) {
         <Typography variant="h1" className={classes.title}>
           {title}
         </Typography>
-        <MilestoneTag color={categoryColor} type={milestoneType} label={milestoneLabel} />
+        {milestoneType && (
```

Comment:
The `milestoneType` should _always_ be present on the ActivityHeader. For an activity template, the milestone type is the `milestone` value in the json. For a milestone, it's the `slug` value. The `categoryType` for a milestone is the `category` in the milestone json.